src/core/dimensions/antimatter-dimension.js

```diff
@@ -323,9 +323,9 @@ export function buyMaxDimension(tier, bulk = Infinity) {
     return;
   }
   let buying = maxBought.quantity;
-  if (buying.gt(bulkLeft)) buying = new Decimal(bulkLeft).times(10);
-  dimension.amount = dimension.amount.plus(buying);
-  dimension.bought = dimension.bought.add(buying);
+  if (buying.gt(bulkLeft)) buying = new Decimal(bulkLeft);
```

Comment:
What does this change, exactly?

Reply:
AD autobuyer will only buy 10% of intended if not doing so
"bulkleft" is set to e100 after maxed AD autobuyer, and the "if" will never trigger since then, and the "times(10)" will never work. The "buying" varible means x times of "buy 10 purchases", so we should multply it by 10.